erigon-lib/downloader/webseed.go

```
@@ -670,7 +670,8 @@ func (d *WebSeeds) callTorrentHttpProvider(ctx context.Context, url *url.URL, fi
 	request = request.WithContext(ctx)
 	resp, err := d.client.Do(request)
 	if err != nil {
-		return nil, fmt.Errorf("webseed.downloadTorrentFile: url=%s, %w", url.String(), err)
+		log.Error("webseed.downloadTorrentFile: ", "url", url.String(), "err", err)
+		return nil, nil
```

Returning nil, nil could lead to silent failures if the caller doesn’t explicitly check for both. In code above I don't see any sense to check for both as returning error is enough.

---

Why you didn't fully remove this check in this case ?
https://github.com/erigontech/erigon/blob/3120de95c5e2add91c7f5d8dff1fd16042f6c4c7/erigon-lib/downloader/webseed.go#L615-L619
  instead of modifying `callTorrentHttpProvider` function
It doesn't feel like a good idea to skip the check 

@mh0lt can you take a look, maybe I am missing something.